### PR TITLE
Add UI for Reset

### PR DESCRIPTION
**Pull request by Lord-Haji**
I am slowly learning console and uhtml

### CODE REVIEW

File: game-cards/SGgame.js [hunk 1]
```diff
@@ -300,6 +300,14 @@ exports.commands = {
 			if (!user.console) return;
 			return this.sendReply('Not Avaliable');
 		},
+		home: function (target, room, user) {
+			if (!user.console) return;
+			user.console.update(false, false, false);
+		},
+		reset: function (target, room, user) {
```
Comment (HoeenCoder) [inline game-cards/SGgame.js]: This should be built into the /playalpha command via aliases. ex: `resetalpha confirmresetalpha`

File: game-cards/SGgame.js [hunk 2]
```diff
@@ -300,6 +300,14 @@ exports.commands = {
 			if (!user.console) return;
 			return this.sendReply('Not Avaliable');
 		},
+		home: function (target, room, user) {
```
Comment (HoeenCoder) [inline game-cards/SGgame.js]: call the command `back`

File: game-cards/SGgame.js [hunk 3]
```diff
@@ -300,6 +300,14 @@ exports.commands = {
...
+			if (!user.console) return;
+			user.console.update(false, false, false);
```
Comment (HoeenCoder) [inline game-cards/SGgame.js]: Theres an issue with this, suppose you were playing the game (the finished version) and you hit reset while on route. when you hit the back button you get the default green and white screen and not the route. To fix this ill shortly be adding `prevScreen` to the console, which tracks the last screen setup. So in preperation of that change this to.

```javascript
user.console.update(this.prevScreen[0], this.prevScreen[1], this.prevScreen[2]);
```

**Review by HoeenCoder — COMMENTED** [review]
Some optimizations:

**Comment by HoeenCoder** [comment]
@Lord-Haji you did not do 2 of 3 of the things i asked you in the review, check the review again.

File: game-cards/SGgame.js [hunk 4]
```diff
@@ -261,15 +264,15 @@ exports.commands = {
...
+				user.console.defaultBottomHTML = '<center><button class="button" name="send" value="/console sound">Toggle Sound</button> <button class="button disabled" name="send" value="/sggame pokedex">Pokedex</button> <button class="button disabled" name="send" value="/sggame pokemon">Pokemon</button> <button class="button disabled" name="send" value="/sggame bag">Bag</button> <button class="button" name="send" value="/sggame pc">PC Boxes</button> <button name="send" value="/search gen7wildpokemonalpha" class="button">Battle!</button> <button name="send" value="/resetalpha" class="button">Reset</button>';
 				user.console.callback = null;
 			};
 			user.console.curText.push('Great choice! I\'ll leave you to your game now.|callback');
 			user.console.init();
 			this.parse('/sggame next');
 		} else {
 			// Continue
-			if (!Db.players.has(user.userid)) return this.parse('/resetalpha');
+			if (!Db.players.has(user.userid)) return this.parse('/confirresetalpha');
```
Comment (HoeenCoder) [inline game-cards/SGgame.js]: `confirmresetalpha`

**Comment by Lord-Haji** [comment]
@HoeenCoder fixed

**Comment by HoeenCoder** [comment]
Will test after I commit my pending changes.

**Comment by Mystifi** [comment]
There are a few typos, especially with the "Are you sure?" display. If you click `No`, it will parse `/confiresetalpha` instead of `/confirmresetalpha`.

**Pull request merged**